model/src/main/java/org/cbioportal/model/MrnaPercentile.java

```diff
@@ -0,0 +1,53 @@
+package org.cbioportal.model;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+
+public class MrnaPercentile implements Serializable {
+
+    private String geneticProfileId;
+    private String sampleId;
+    private Integer entrezGeneId;
+    private BigDecimal percentile;
+    private BigDecimal zScore;
+
+    public String getGeneticProfileId() {
+        return geneticProfileId;
+    }
+
+    public void setGeneticProfileId(String geneticProfileId) {
+        this.geneticProfileId = geneticProfileId;
+    }
+
+    public String getSampleId() {
+        return sampleId;
+    }
+
+    public void setSampleId(String sampleId) {
+        this.sampleId = sampleId;
+    }
+
+    public Integer getEntrezGeneId() {
+        return entrezGeneId;
+    }
+
+    public void setEntrezGeneId(Integer entrezGeneId) {
+        this.entrezGeneId = entrezGeneId;
+    }
+
+    public BigDecimal getPercentile() {
+        return percentile;
+    }
+
+    public void setPercentile(BigDecimal percentile) {
+        this.percentile = percentile;
+    }
+
+    public BigDecimal getzScore() {
```

Comment:
In the future, we may remove all zscores profiles in the database and implement frontend zscore calculation. Maybe for the mRNA percentile feature, it's OK not to send back zScores. The percentile is probably enough.

@ecerami are you OK if we remove the `mRNA z-score` line below and also remove it from the api?

![image](https://cloud.githubusercontent.com/assets/840895/21946042/9b483c12-d9ac-11e6-892f-ca97971340ad.png)